lib/tasks/import.rake

```diff
@@ -36,4 +36,62 @@ namespace :import do
       end
     end
   end
+
+  desc "Updates items with categories"
+  task category_csv: :environment do
+
+    class Category
+      def refresh_category_nodes
+        # no need to do this after every change
```

Comment:
This is a hack to prevent the materialized view from being refreshed after every categorization change. Without this, the import would be a lot slower.